### PR TITLE
ci: disable swagger check temporarily

### PR DESCRIPTION
**Pull request by 0Tech**
## Description
Checking generated swagger files (#457) is not working as expected, so I remove the logic and insert an action to print the log for the analysis.

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes. [no need]
- [ ] I have updated the documentation accordingly. [no need]
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml` [no need]

### CODE REVIEW

**Review by dudong2 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -50,5 +50,6 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * (ci) [\#457](https://github.com/line/lbm-sdk/pull/457) add swagger check
 * (ci) [\#469](https://github.com/line/lbm-sdk/pull/469) publish docker image on tag push
+* (ci) [\#471](https://github.com/line/lbm-sdk/pull/471) disable swagger check temporarily
```
Comment (zemyblue) [inline CHANGELOG.md:53]: I think this changes is a fix for #457. 
Then, how about adding it to line 51 like below?
```
* (ci) #457, #471 add swagger check
```

Comment (0Tech) [reply CHANGELOG.md:53]: I agree on squashing the entries as you suggested. But the check would not working after this PR. Let me change the title.

Comment (0Tech) [reply CHANGELOG.md:53]: Is it OK to have different title with it's PR's? Or I cannot change the title because it affects the previous PR (#457).

**Review by zemyblue — APPROVED** [review]
LGTM

**Pull request merged**